sdk/spring/README.md

```diff
@@ -19,23 +19,37 @@ You can use the following command to install jars into your local repository qui
 
 ```shell
 mvn clean install \
-  -f sdk/spring/ \
+  -Dcheckstyle.skip=true \
+  -Dcodesnippet.skip \
+  -Denforcer.skip \
+  -Djacoco.skip=true \
+  -Dmaven.javadoc.skip=true \
+  -Drevapi.skip=true \
+  -DskipTests \
```

Comment:
why do users need the test jar when they install the libraries for local development?

Reply:
Already answered here: https://github.com/Azure/azure-sdk-for-java/pull/27992#discussion_r840278235